### PR TITLE
fix: moved chared zod schemas, updated password generation for tests

### LINKED ISSUE #1336: [Subtask] -Share types for Auth
We need to check if there are shared zod schemas and return types between frontend and backend in Test Wallet.
If there are, we should move them to the shared package.

### PR DESCRIPTION
**Pull request by adrianboros**
### Context

- fixes #1336 

### Changes
- moved chared zod schemas, updated password generation for tests

### CODE REVIEW

File: packages/wallet/backend/src/rafiki/validation.ts [hunk 1]
```diff
@@ -1,15 +1,6 @@
 import { z } from 'zod'
 import { EventType, PaymentType } from './service'
 
-export const ratesSchema = z.object({
```
Comment (dragosp1011) [inline packages/wallet/backend/src/rafiki/validation.ts:4]: rate endpoint is used by rafiki and not wallet frontend, it should not be moved in shared

File: packages/wallet/backend/src/rapyd/schemas.ts [hunk 2]
```diff
@@ -37,15 +37,6 @@ export const walletSchema = z.object({
   })
 })
 
-export const ratesSchema = z.object({
```
Comment (dragosp1011) [inline packages/wallet/backend/src/rapyd/schemas.ts:40]: schemas for rapyd should not be moved

File: packages/wallet/backend/src/rafiki/validation.ts (hunk 1 above)
Comment (adrianboros) [reply packages/wallet/backend/src/rafiki/validation.ts:4]: removed from here and use this definition from rates/validation

File: packages/wallet/backend/src/rapyd/schemas.ts (hunk 2 above)
Comment (adrianboros) [reply packages/wallet/backend/src/rapyd/schemas.ts:40]: removed from here and use this definition from rates/validation

**Review by dragosp1011 — APPROVED** [review]

**Review by dragosp1011 — APPROVED** [review]

**Pull request merged**